lib/dry/struct/class_interface.rb

```diff
@@ -30,7 +31,7 @@ def inherited(klass)
       # @yield
       #   If a block is given, it will be evaluated in the context of
       #   and new struct class, and set as a nested type for the given
-      #   name
+      #   name, a constant will also be defined for the nested type
```

Comment:
maybe we could reword this to:

> If a block is given, it will be evaluated in the context of a new struct class, and set as a nested type for the given attribute. A class with a matching name will also be defined for the nested type.

Does that make things a little clearer?

Reply:
Updated